apps/oc_chef_authz/src/oc_chef_policy_group.erl

```diff
@@ -0,0 +1,180 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
```

Comment:
The policy stuff got implemented in oc_chef_authz for reasons I don't know well enough to explain back to you, so I'm just going with the flow. My understanding is that oc_chef_authz needs to get exploded and have most of its functionality moved into chef_db and chef_objects which is supposed to happen sometime in the future.